### DISCUSSION TITLE
1.5.2 are *.js files anywhere?

### DISCUSSION THREAD

**Discussion by bilo1967** [peers/peerjs#1181] [General]
I got lost with the 1.5.2 release: are peerjs.js, peerjs.min.js and relative maps anywhere?

**Comment by jonasgloning** [comment]
Hey @bilo1967, they are on 

- [NPM](https://www.npmjs.com/package/peerjs?activeTab=code)
- [UNPKG](https://unpkg.com/peerjs@1.5.2/dist/)
- ESM.sh / [jsDelivr](https://www.jsdelivr.com/package/npm/peerjs) / etc.

Where were you expecting them? Should I upload them to somewhere else?
